index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Build a Tribute Page</title>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main">
+       <h1 id="title">Dr. Norman Borlaug</h1>
+       <figure id="img-div">
+         <img id="image" src="img/img.jpeg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field"/>
+         <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+       </figure>
+       <section id="tribute-info">
+        <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a>
+       </section>
+    </main>
```

Comment:
How about renaming the main tag from lines 12 and 21 to a section tag, this will helps further showcasing that the content inside is grouped (i.e. relates to a single theme), and should appear as an entry in an outline of the page.
The whole of this should be a section tag". 🤗

Meanwhile when using the main content area, it indicates that the area consists of content that is directly related to or expands upon the central topic of a document, or the central functionality of an application. 👏